src/controllers/app/userController.js

```diff
@@ -95,3 +90,42 @@ exports.CheckCode = function (req, res) {
     }
   });
 };
+
+exports.completeSignup = function (req, res) {
+  const { operationId, userName, password } = req.body;
+  var { userId, code } = jwt.verify(operationId, process.env.JWT_SECRET_KEY);
+  console.log("userId", userId);
+  try {
+    User.findById(userId, (err, foundedUser) => {
+      if (err) {
+        return res.status(500).json({ message: "server error" });
+      }
+      if (foundedUser === null) {
+        return res.status(404).json({ message: "user not found" });
+      } else {
+        if (foundedUser.code.text !== code) {
+          return res.status(400).json({ message: "wrong enteries" });
+        }
+        if (foundedUser.isActive) {
+          return res.status(400).json({ message: "user is already active" });
+        }
+        //updated inActive user
+        updatedUser = foundedUser;

```

Comment:
Where did you define updatedUser variable?

Reply:
done